src/Runner.jl

```diff
@@ -241,6 +241,22 @@ function generate_compiler_wrappers!(platform::AbstractPlatform; bin_path::Abstr
         return flags
     end
 
+    function min_macos_version_flag(p::AbstractPlatform)
+        # If no `os_version` is specified in `p`, default to the oldest we support in the Julia world,
+        # which is `10.8`, but if it is actually specified, then set that corresponding value.
+        #min_macos_version = something(os_version(p), v"14.0.0")
+
+        # Eventually, we'll take this in `os_version(p)`, but not just yet.  We need to fix the paths
+        # to the compiler shards first, since right now they have `14` at the end
+        min_macos_version = v"14.0.0"
+
+        # `os_version` stores the kernel version, we translate that here to 10.X version:
+        min_macos_version = min_macos_version.major - 6
+
+        # Always ask for a minimum macOS version of 10.8, as is default for the whole Julia world
+        return "-mmacosx-version-min=10.$(min_macos_version)"
```

Comment:
This is
```
"-mmacosx-version-min=10.8.0.0"
```
is that correct?

Reply:
`min_macos_version` is not a VersionNumber here, it's an Int.

Reply:
Uh uh, you're right